designer/server/src/main/scala/pl/touk/nussknacker/ui/util/AkkaToTapirStreamExtension.scala

```diff
@@ -0,0 +1,53 @@
+package pl.touk.nussknacker.ui.util
```

Comment:
I don't like the general purpose util package concept. I think that we should keep it in the place near to the server logic handling, maybe `pl.touk.nussknacker.ui.server`? WDYT?